src/components/NavBar/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import { BeaconContext } from '../../context/beacon-context';
+import { Link, navigate } from 'gatsby';
+import Theme from 'src/assets/theme.svg';
+import userAtom from '../../atoms/user-atom';
+import isUserAtom from '../../atoms/is-user-atom';
+import { useAtom } from 'jotai';
+import { createUser } from '../../api';
+import { setUserId } from 'amplitude-js';
```

Comment:
@manangouhari Please try removing the `amplitude-js` import.

![Screenshot 2021-01-30 at 12 51 59 PM](https://user-images.githubusercontent.com/16613455/106350210-0e86e680-62fa-11eb-964d-492a2d429f71.png)

Since amplitude is also using window object under the hood, that's why it's resulting in a build error and it's not being used here anyways.

Reply:
Tried removing it & run a local build. It's working, yes!!!!

Reply:
Damn. How do I keep missing these small things. Need to up my game.